app/uk/gov/hmrc/taxenrolmentassignmentfrontend/AccountTypes.scala

```diff
@@ -19,14 +19,13 @@ package uk.gov.hmrc.taxenrolmentassignmentfrontend
 import play.api.libs.json.Reads
 
 object AccountTypes extends Enumeration {
-  val SINGLE_ACCOUNT: AccountTypes.Value = Value("SINGLE_ACCOUNT")
   val PT_ASSIGNED_TO_CURRENT_USER: AccountTypes.Value = Value(
     "PT_ASSIGNED_TO_CURRENT_USER"
   )
   val PT_ASSIGNED_TO_OTHER_USER: AccountTypes.Value = Value(
     "PT_ASSIGNED_TO_OTHER_USER"
   )
-  val MULTIPLE_ACCOUNTS: AccountTypes.Value = Value("MULTIPLE_ACCOUNTS")
+  val SINGLE_OR_MULTIPLE_ACCOUNTS: AccountTypes.Value = Value("SINGLE_OR_MULTIPLE_ACCOUNTS")
```

Comment:
Without the call to IV and if the nino has no SA or HMRC-PT enrolment we don't know anymore if the user has multiple creds or not.